cryptofeed/rest/kraken.py

```diff
@@ -0,0 +1,279 @@
+from time import time
+import hashlib, hmac, requests, urllib, json, base64
+
+from cryptofeed.rest.api import API
+from cryptofeed.feeds import KRAKEN
```

Comment:
you should import it from cryptofeed.exchanges since thats the place they are all defined